cmd/influxd/backup/backup.go

```diff
@@ -88,32 +81,28 @@ func (cmd *BackupCommand) Run(args ...string) error {
 }
 
 // parseFlags parses and validates the command line arguments.
-func (cmd *BackupCommand) parseFlags(args []string) (url.URL, string, error) {
+func (cmd *Command) parseFlags(args []string) (host string, path string, err error) {
 	fs := flag.NewFlagSet("", flag.ContinueOnError)
-	host := fs.String("host", DefaultSnapshotURL.String(), "")
+	fs.StringVar(&host, "host", "localhost:8088", "")
 	fs.SetOutput(cmd.Stderr)
 	fs.Usage = cmd.printUsage
 	if err := fs.Parse(args); err != nil {
-		return url.URL{}, "", err
-	}
-
-	// Parse host.
-	u, err := url.Parse(*host)
-	if err != nil {
-		return url.URL{}, "", fmt.Errorf("parse host url: %s", err)
+		return "", "", err
 	}
 
-	// Require output path.
-	path := fs.Arg(0)
-	if path == "" {
-		return url.URL{}, "", fmt.Errorf("snapshot path required")
+	// Ensure that only one arg is specified.
+	if fs.NArg() == 0 {
```

Comment:
Minor -- this could be combined into a single check, if the error message was "a single snapshot path is required". This is purely a style issue.